generate-xml/domain.nix

```diff
@@ -386,7 +386,13 @@ let
                   (subelem "backend" [ (subattr "model" typeString) ] (sub "source" typePath))
                   addresselem
                 ])
-              (subelem "memballoon" [ (subattr "model" typeString) ] [ addresselem ])
+              (subelem "memballoon"
+                [
+                  (subattr "model" typeString)
+                  (subattr "autodeflate" typeBoolYesNo)
+                  (subattr "freePageReporting" typeBoolYesNo)
```

Comment:
Looks like `autodeflate` and `freePageReporting` should be `typeBoolOnOff` rather than `typeBoolYesNo`?

Reply:
Ah, you're right.